diff --git a/profiles/contrib/config_installer/src/Form/SyncConfigureForm.php b/profiles/contrib/config_installer/src/Form/SyncConfigureForm.php
index 7ca4099..6306875 100644
--- a/profiles/contrib/config_installer/src/Form/SyncConfigureForm.php
+++ b/profiles/contrib/config_installer/src/Form/SyncConfigureForm.php
@@ -38,7 +38,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $form['sync_directory'] = [
       '#type' => 'textfield',
       '#title' => $this->t('Synchronisation directory'),
-      '#default_value' => config_get_config_directory(CONFIG_SYNC_DIRECTORY),
+      '#default_value' => 'sites/default/config/sync', // config_get_config_directory(CONFIG_SYNC_DIRECTORY),
       '#maxlength' => 255,
       '#description' => $this->t('Path to the config directory you wish to import, can be relative to document root or an absolute path.'),
       '#required' => TRUE,
